src/chigame/games/urls.py

```diff
@@ -3,5 +3,7 @@
 from . import views
 
 urlpatterns = [
-    path("", views.GameListView.as_view(), name="games-list"),
+    path("", views.GameListView.as_view(), name="game-list"),
+    path("create/", views.GameCreateView.as_view(), name="game-create"),
+    path("<int:pk>/edit", views.GameEditView.as_view(), name="game-edit"),
```

Comment:
Good namespaced names. Continue this format of `game-` for your urls as it will maximally avoid URL conflict (where someone else gave the same name to their URL as one of yours)